### PR TITLE
only use the required lodash components

### PR DESCRIPTION
**Pull request by eddiemonge**
Only include the lodash components that are actually used

Builds on #5 and improves it

### CODE REVIEW

**Comment by EmilioAiolfi** [comment]
@finneganh this PR not resolve the issue [link](https://github.com/twitter-fabric/velocity-react/issues/77) ?

**Comment by klaytonfaria** [comment]
@EmilioAiolfi and @eddiemonge It seems to solve the issue

File: package.json [hunk 1]
```diff
@@ -24,7 +24,16 @@
...
+    "lodash.assign": "^4.2.0",
+    "lodash.defaults": "^4.2.0",
+    "lodash.foreach": "^4.5.0",
+    "lodash.isequal": "^4.5.0",
+    "lodash.isobject": "^3.0.2",
+    "lodash.keys": "^4.2.0",
+    "lodash.map": "^4.6.0",
+    "lodash.omit": "^4.5.0",
+    "lodash.random": "^3.2.0",
+    "lodash.sample": "^4.2.1",
```
Comment (klaytonfaria) [inline package.json]: Why not use fixed version? I think would be safe to prevent new issues.

**Comment by finneganh** [comment]
Pretty sure that this wouldn't solve any bundle size issues as now the 4.x lodash modules may be coming from different places. 

I maintain that in a correctly-functioning build setup the current use of lodash 3 should not cause a problem.

**Pull request closed without merging**